tests/RME/gold_hrrr/config.ini

```diff
@@ -41,23 +42,21 @@ time_zone:                     UTC
 ################################################################################
 
 [gridded]
-directory:                     /home/micahsandusky/Code/smrf/tests/RME/gridded/hrrr_test
-n_forecast_hours:              18        
-forecast_flag:                 False     
-zone_letter:                   N         
 data_type:                     hrrr      
+directory:                     /home/scotthavens/code/smrf/tests/RME/gridded/hrrr_test
```

Comment:
Lets changes the paths back to relative to avoid user names.

Reply:
Removing